File: 01. Life, The Universe And Everything/Solution.py
"""
	STATS:
		Runtime - 0.02
		Memory - 9.2 MB
		Language - Python 3.7.3
"""

if __name__ == '__main__':
	# Running an infinite loop
	while True:
		# Taking input from the user and type-casting it into integer
		number = int(input())
		
		# Checking, if `number` is 42, breaking out of the loop
		if number == 42:
			break
	
		# If the number is not 42, printing the number as it is.
		print(number)

File: 01. Life, The Universe And Everything/Solution.cs
/**
  * STATS:
  * 	Runtime - 0.02
  *		Memory - 16 MB
  *		Language - C# (GMCS 5.20.1)
  */

using System;

public class Solution {
	public static void Main() {
		// Running an infinite loop
		while (true)	{
			// Taking input from the console as a line, and converting it into integer
			int input = Convert.ToInt32(Console.ReadLine());
			
			// Checking, if the number is 42, breaking out of the infinite loop.
			if (input == 42)
				break;
			
			// If the number isn't 42, printing the number to the console.
			Console.WriteLine(input);
		}
	}
}